website/docs/admin_hosts_houdini.md

```diff
@@ -3,9 +3,26 @@ id: admin_hosts_houdini
 title: Houdini
 sidebar_label: Houdini
 ---
+## General Settings
+### JOB Path
+you can add your studios preffered JOB Path, JOB value will be checked and updated on file save and open.
+Disableing this option will effectivly turn off this feature.
+
+JOB Path can be:
+- Arbitrary hardcoded path
```

Comment:
I think `hardcoded` is not necessary here, since it's implied we are setting it to a path